NEWS.md

```diff
@@ -3,6 +3,204 @@ XGBoost Change Log
 
 This file records the changes in xgboost library in reverse chronological order.
 
+## 2.0.0 (2023 Aug 16)
+
+We are excited to announce the 2.0 release for XGBoost. In this release note, we will first walk through some of the general changes then introduce package-specific updates.
+
+### Initial work on multi-target tree
+In 2.0, we started initial work on vector-leaf tree models for multi-target regression, multi-label classification, and multi-class classification. Previously, XGBoost builds one model for each target, now with this working-in-progress feature, XGBoost can build one tree for all targets. The feature has multiple benefits and trade-offs compared to the existing method, it can help prevent overfitting, produce smaller models, and build trees that in theory consider the correlation between targets. In addition, users can mix two types of trees (vector leaf and scalar leaf) in a training session with a callback. Please note that the feature is still working in progress and most features are missing. See #9043 for current status. Related PRs: (#8538, #8697, #8902, #8884, #8895, #8898, #8612, #8652, #8698, #8908, #8928, #8968, #8616, #8922, #8890, #8872, #8889) Please note that, only the `hist` (default) tree method on CPU can be used for building vector leaf trees at the moment.
+
+### New `device` parameter.
+
+A new `device` parameter is set to replace the existing `gpu_id`, `gpu_hist`, `gpu_predictor`, and the PySpark specific parameter `use_gpu`. Onward, users need only the `device` parameter to select which device to run along with the ordinal of the device. For more information, please see our document page (https://xgboost.readthedocs.io/en/stable/parameter.html#general-parameters) . For example, with  `device="cuda", tree_method="hist"`, XGBoost will run the `hist` tree method on GPU. (#9363, #8528, #8604, #9354, #9274, #9243, #8896, #9129, #9362, #9402, #9385, #9398, #9390, #9386, #9412). The old behavior of ``gpu_hist``  is preserved but deprecated. In addition, the `predictor` parameter is removed.
+
+
+### `hist` is now the default tree method
+Starting from 2.0, the `hist` tree method will be the default. In previous versions, XGBoost chooses `approx` or `exact` depending on the input data and training environment. The new default can help XGBoost train models more efficiently and be more consistent. (#9320, #9353)
+
+### GPU-based approx tree method
+There's initial support for using the `approx` tree method on GPU. The performance of the `approx` is not yet well optimized but is feature complete. It can be accessed through the use of parameter combination `device="cuda", tree_method="approx"`. (#9414, #9399, #9478). Please note that Scala-based Spark interface is not yet supported.
```

Comment:
```suggestion
There's initial support for using the `approx` tree method on GPU. The performance of the `approx` is not yet well optimized but is feature complete except for JVM packages. It can be accessed through the use of the parameter combination `device="cuda", tree_method="approx"`. (#9414, #9399, #9478). Please note that the Scala-based Spark interface is not yet supported.
```